pkg/api/stream.go

```diff
@@ -206,7 +206,7 @@ type StreamContext interface {
 	TransformOutput(data interface{}) ([]byte, bool, error)
 	// Decode is set in the source according to the format.
 	// It decodes byte array into map or map slice.
-	Decode(data []byte) (map[string]interface{}, error)
+	Decode(data []byte) ([]map[string]interface{}, error)
```

Comment:
This is a breaking change in API. We are trying to avoid such changes in the 1.x version. We may only introduce such changes in v2.